alphabase/spectral_library/base.py

```diff
@@ -351,53 +351,70 @@ def calc_precursor_isotope_intensity(self,
         """
 
         if 'precursor_mz' not in self._precursor_df.columns:
-            self.calc_precursor_mz()
-            self.clip_by_precursor_mz_()
+            self.calc_and_clip_precursor_mz()
 
-        if multiprocessing and len(self.precursor_df)>mp_batch_size:
+        if mp_process_num>1 and len(self.precursor_df)>mp_batch_size:
             (
                 self._precursor_df
             ) = precursor.calc_precursor_isotope_intensity_mp(
                 self.precursor_df, 
                 max_isotope = max_isotope,
                 min_right_most_intensity = min_right_most_intensity,
+                normalize=normalize,
                 mp_process_num = mp_process_num,
+                mp_batch_size=mp_batch_size,
             )
         else:
             (
                 self._precursor_df
             ) = precursor.calc_precursor_isotope_intensity(
                 self.precursor_df, 
                 max_isotope = max_isotope,
+                normalize=normalize,
                 min_right_most_intensity = min_right_most_intensity,
             )
-            
+
+    def calc_precursor_isotope(self,
```

Comment:
If you like we can keep only one `calc_precursor_isotope_intensity` or `calc_precursor_isotope` so there's no redundancy.

Reply:
`calc_precursor_isotope` is now `calc_precursor_isotope_info`. My original thought is that maybe it is useful in AlphaDIA, but I think your `calc_*_intensity` is better in readability and memory-saving.